src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -22,14 +21,11 @@ public unsafe class ExpirationStorage
         private const string DocumentsByRefresh = "DocumentsByRefresh";
 
         private readonly DocumentDatabase _database;
-        private readonly DocumentsStorage _documentsStorage;
-        private readonly Logger _logger;
+        private static readonly Size MaxTransactionSize = new(16, SizeUnit.Megabytes);
```

Comment:
Do we need it to be fixed? Or should we have some advanced configuration option for this?

Reply:
If that could be not hard coded then maybe we can adjust it in the tests. Can we have some tests for this PR?

Reply:
Moved and added a test